### DISCUSSION TITLE
Comment fonctionne une liste de choix déroulante ?

### DISCUSSION THREAD

**Discussion by Mathis554** [witsa/synapps#748] [Q&A]
Bonjour,
Je n'arrive pas à comprendre comment fonctionne l'acteur liste de choix déroulante malgré le fait d'avoir lu la [documentation](https://witsa.github.io/synapps/concepts/actor-types/input-dropdown-list.html).
J'aurais aimé pouvoir sélectionner l'un de mes compteurs et pouvoir lire sa valeur.

**Comment by WITNicolas** [comment]
Bonjour,
La boite deroulante permet d'afficher un texte et de lui associer une valeur de votre choix. nombre/chaine

**Reply by Mathis554** [reply]
Bonjours,
Comment je fait pour associer des valeurs à mes choix ?

**Reply by WITNicolas** [reply]
Bonjour,

Il suffit d'editer les options de la liste déroulante
![image](https://github.com/user-attachments/assets/4a804a3d-9ba5-40da-9f08-b5ff85c5d01c)

Et donc pour chaque option il y aura un texte (ce qui sera affiché) et une valeur (ce qui sera exploité)
![image](https://github.com/user-attachments/assets/6e51758e-3816-4237-8142-09575277896e)

[Lien vers la documentation Dropdown-List](https://witsa.github.io/synapps/concepts/actor-types/input-dropdown-list.html)

**Reply by Mathis554** [reply]
Bonjours
Il est possible de le lier à une valeur qui est dans mon ULI

**Reply by WITNicolas** [reply]
Bonjour,

Biensur, en utilisant la propriété **SELECTION**, en faisant une liaison avec une variable du REDY (ne pas oublier de cocher ECRITURE)
![image](https://github.com/user-attachments/assets/a9c3fff7-5552-472e-850e-fc45c49ca4f1)

**Reply by Mathis554** [reply]
Cette solution ne me permet pas d'afficher des valeurs indépendantes par choix. Admettions que je sélection compteurs 1, je voudrais avoir sa valeur et quand je sélectionne compteurs 2 avoir la valeur de compteurs 2.
